hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/kafka/KafkaMessageReceiver.java

```diff
@@ -49,33 +45,27 @@ public KafkaMessageReceiver(String topicName, ConsumerConnector consumerConnecto
     }
 
     @Override
-    public RawMessage next() {
+    public Message next() {
+        Timer.Context readingTimerContext = readingTimer.time();
```

Comment:
`Timer.Context` can be managed via try-with-resources. It implements `AutoCloseable`.